cloud-deploy/src/utils/cloud-deploy.schema.json

```diff
@@ -393,6 +393,11 @@
               "type": "string",
               "pattern": "^[a-z]{3}$"
             },
+            "auth-proxy": {
+              "description": "Auth proxy to use",
+              "type": "string",
+              "enum": ["envoy-opa", "none"]
```

Comment:
Add `"default": "envoy-opa"` in schema for documentation purposes.

Reply:
Added